Mathlib/Topology/PartitionOfUnity.lean

```diff
@@ -665,15 +665,4 @@ theorem exists_continuous_sum_one_of_isOpen_isCompact [T2Space X] [LocallyCompac
           exact hj)]
     rfl
   intro i x
-  refine ⟨f.nonneg i x, ?_⟩
-  by_cases h0 : f i x = 0
-  · rw [h0]
-    exact zero_le_one
-  rw [← Finset.sum_singleton (f ·  x) i]
-  apply le_trans _ (f.sum_le_one' x)
-  rw [finsum_eq_sum (f.toFun ·  x) (by exact toFinite (support (f.toFun · x)))]
-  simp only [Finite.toFinset_setOf, ne_eq]
-  gcongr with z hz
-  · exact fun j _ _ => f.nonneg j x
-  simp only [Finset.singleton_subset_iff, Finset.mem_filter, Finset.mem_univ, true_and]
-  exact h0
+  exact ⟨f.nonneg i x, PartitionOfUnity.le_one f i x⟩
```

Comment:
The type required for the second component here is `(f i) x ≤ 1`, which is exactly what `PartitionOfUnity.le_one` gives us. (No defeq shenanigans.)